P5/Source/Guidelines/en/TD-DocumentationElements.xml

```diff
@@ -858,35 +859,39 @@ to mark any technical term, thus:
           </constraintSpec>
         </egXML>
       </p>
-      <p>The constraints in the preceding example all related to attributes in the empty namespace, and the
-        Schematron rules did not therefore need to define a TEI namespace prefix. The Schematron language <gi
-          scheme="Schematron">ns</gi> element should be used to do this when a constraint needs to refer to a
-        TEI element, as in the following example, which models the constraint that a TEI <gi>div</gi> must
-        contain either no subdivisions or at least two of them:
+      <p>Note that the <q>tei:</q> prefix needs to be bound to the TEI
+      namespace using the Schematron language <gi
+      scheme="Schematron">ns</gi> element, but typically this only
+      need be done once. The following two examples are written
```

Comment:
Oooh, good question. I don’t think so. I think — but am not at all sure — that “need be done” is sort of an idiomatic way of saying either “needs to be done” or “needs to have been done”. In any case, I do not think I have ever heard or seen “needs be done”.

But in any case, perhaps the best thing to do is to avoid this usage altogether and write “… typically this only needs to be done once”.

Reply:
This is a good point. I read it first thinking it was fine but after you've pointed this out, I'm not certain. I concur with writing "... typically this only needs to be done once".

Reply:
I'm correcting this now (while testing locally) to "this only needs to be done once."